website/docs/d/clusters.html.markdown

```diff
@@ -79,8 +79,8 @@ In addition to all arguments above, the following attributes are exported:
     - DELETED
     - REPAIRING
 * `auto_scaling_disk_gb_enabled` - Indicates whether disk auto-scaling is enabled.
-* `auto_scaling_compute_enabled` - (Optional) Specifies whether cluster tier auto-scaling is enabled. The default is false.
-* `auto_scaling_compute_scale_down_enabled` - (Optional) Set to `true` to enable the cluster tier to scale down.
+* `auto_scaling_compute_enabled` - Specifies whether cluster tier auto-scaling is enabled. The default is false.
+* `auto_scaling_compute_scale_down_enabled` -  Set to `true` to enable the cluster tier to scale down.
```

Comment:
```suggestion
* `auto_scaling_compute_scale_down_enabled` - Specifies whether cluster tier auto-down-scaling is enabled.
```

Reply:
documentation fixed!.

the `advanced_cluster` schema is a bit different from `cluster` regarding auto scaling properties and it seems that works correctly